src/babel.js

```diff
@@ -44,11 +44,7 @@ exports.getCachePath = function (sourceCode) {
 exports.compile = function (sourceCode, filePath) {
   if (!babel) {
     babel = require('babel-core')
-    var Logger = require('babel-core/lib/transformation/file/logger')
-    var noop = function () {}
-    Logger.prototype.debug = noop
-    Logger.prototype.verbose = noop
-
+    require('debug').log = () => { } // Stop Babel logging to stdout/stderr (would error when no console attached)
```

Comment:
I might be completely wrong about how Node modules work, but won't this affect `debug.log` **everywhere**? 
Pretty sure the module cache is not scoped.